Rubberduck.Core/UI/Command/MenuItems/CommandBars/SerializeProjectsCommandMenuItem.cs

```diff
@@ -76,6 +76,16 @@ protected override void OnExecute(object parameter)
                 Logger.Warn($"Serializing {library.Path}.");
                 _serializationProvider.SerializeProject(library);
             }
+
+#if DEBUG
```

Comment:
Isn't the precompiler directive redundant here?

Reply:
AFAICT, this file doesn't have any other `#DEBUG` anywhere else. Besides, given that the `Serialize` method can only exist in a `DEBUG` build, it makes sense to wrap the invocation inside a `DEBUG` block to ensure it doesn't end up calling a non-existent method.

Reply:
My thought when I suggested that was that the command only gets injected in DEBUG. I missed the fact that `Serialize` is also conditionally on the interface.